reana_commons/config.py

```diff
@@ -364,6 +364,17 @@ def kubernetes_node_label_to_dict(node_label):
 cluster creation time.
 """
 
+
+K8S_USE_SECURITY_CONTEXT = (
+    os.getenv("K8S_USE_SECURITY_CONTEXT", "True").lower() == "true"
+)
```

Comment:
Wouldn't just something like this be enough?

```python
K8S_USE_SECURITY_CONTEXT = os.getenv("K8S_USE_SECURITY_CONTEXT", True)
```

Reply:
Environment variables are strings.  The extra code both converts to boolean and makes sure that we capture "True", "true", "TRUE", and so on.   If a user does `export K8S_USE_SECURITY_CONTEXT=False`, that might be confusing.

(That being said, I don't feel strongly about this).